### DISCUSSION TITLE
Будет ли релиз VKUI компонентов под Vue?

### DISCUSSION THREAD

**Discussion by horanchikk** [VKCOM/VKUI#2667] [Q&A] [answered]
title

**Comment by SevereCloud** [comment, marked as answer]
Портирование большой библиотеки на другой фреймворк это трудоемкий процесс. На поддержание и синхронизацию кодовой базы также необходимо время

VKUI распространяется под лицензией [MIT](https://github.com/VKCOM/VKUI/blob/master/LICENSE), так что любой человек может [портировать](https://github.com/search?q=VKUI+vue) библиотеку, лицензия это позволяет
